### PR TITLE
Fix usage message

### PR DESCRIPTION
**Pull request by yuawn**
Fix afl-fuzz usage message overflow and sentence.

### CODE REVIEW

File: src/afl-fuzz.c [hunk 1]
```diff
@@ -383,9 +383,9 @@ static int stricmp(char const *a, char const *b) {
 static void fasan_check_afl_preload(char *afl_preload) {
 
   char   first_preload[PATH_MAX + 1] = {0};
-  char  *separator = strchr(afl_preload, ':');
+  char * separator = strchr(afl_preload, ':');
```
Comment (domenukk) [inline src/afl-fuzz.c]: What version of clang-format do you have?

File: src/afl-fuzz.c [hunk 2]
```diff
@@ -167,8 +167,8 @@ static void usage(u8 *argv0, int more_help) {
       "                  See docs/README.MOpt.md\n"
       "  -c program    - enable CmpLog by specifying a binary compiled for "
       "it.\n"
-      "                  if using QEMU/FRIDA or if you the fuzzing target is "
-      "compiled"
+      "                  if using QEMU/FRIDA or the fuzzing target is "
+      "compiled\n"
```
Comment (domenukk) [inline src/afl-fuzz.c:171]: Wouldn't the terminal automatically wrap this line?

File: src/afl-fuzz.c (hunk 1 above)
Comment (yuawn) [reply src/afl-fuzz.c]: I used clang-format-10, code-format again with clang-format-14

File: src/afl-fuzz.c (hunk 2 above)
Comment (yuawn) [reply src/afl-fuzz.c:171]: On Mac terminal
Screen Shot 2022-07-24 at 6 41 18 PM

**Comment by vanhauser-thc** [comment]
thank you!

**Pull request merged**